### ISSUE TITLE
cid_num in table sccpline does not fill in when creating SCCP extension

### ISSUE THREAD

**Issue by metanot** [chan-sccp/sccp_manager#74]
Sccp Manager v14.5.0.4

At some point all newly created extensions does not have cid_num filled with extension number, therefore all internal calls from that extensions shows only info from cid_name column

(In fact it's a rather useful for cases, where i need to hide extension number in incoming calls, but i think FreePBX can handle such cases with other tools :) )

Here is a screenshot without filled cid_num
![sccpline_w_error](https://user-images.githubusercontent.com/58426122/154236099-a7e37dc2-c278-4a9f-8578-6ebcbe156a5f.png)

But, if i fill in Outbound CID - cid_num is correctly populating with my Outbound CID
![sccpline_wo_error](https://user-images.githubusercontent.com/58426122/154236345-8a95d369-12c7-437d-94dd-ec00cc77d52d.png)

And, if i remove Outbound CID for extension, cid_num is reverting to NULL

**Comment by carlsondan** [comment]
Yes, have a look at Sccp.class.php. I'm not sure why this is being done, but effectively, the code is taking what is being populated in the outboundcid field and assigning it to the cid_num field in the sccpline table. If you add a new line and the outboundcid field is left blank then the cid_num field in the sccpline table is given a null value. This causes problems for internal calling and for accessing voice mail (*97), where it does not recognize the extension. 

The workaround I did involved removing the code that assigned outboundcid to $settings['cid_num']['value'], modified code to parse outboundcid if it included cid name and number and assign number to $outboundcid only. I then assigned a the $id value (the extension number) to  $settings['cid_num']['value'].

After making these changes, doing a "sccp show line XXXX" shows both the "Caller ID name" and the "Caller ID number" properly. Doing a "database show ampuser XXXX" shows the correct information in /AMPUSER/XXXX/cidname, /AMPUSER/XXX/cidnum, and /AMPUSER/XXXX/outboundcid. Putting a value in the "Outbound CID" form field only changes /AMPUSER/XXXX/outboundcid as I believe it should. More importantly, doin an internal call display the correct cid_name and cid_number on the receiving end, and when the user dials voice mail (*97), it recognizes the calling extension. If an oubound call is made and value was put into the "Outbound CID" form field, then that override value is sent out the trunk line.
